packages/reports/src/Containers/open-positions.tsx

```diff
@@ -80,18 +81,19 @@ type TMobileRowRenderer = TUiStore & {
 };
 
 type TOpenPositionsTable = {
+    accumulator_rate: string;
     className: string;
     columns: Record<string, unknown>[];
     component_icon: string;
+    contract_type_value: string;
     currency: string;
-    active_positions: TPortfolioStore['active_positions'];
```

Comment:
@akmal-deriv  circle ci checks failed, please check

Reply:
Sure, I noticed that